### DISCUSSION TITLE
multiple embedded mode instances

### DISCUSSION THREAD

**Discussion by psanthoshk** [olric-data/olric#188] [Q&A] [answered]
To evaluate I want to run multiple instances in embedded mode, any sample implementation for the same ?
  
  // Code snippet from samples
  dm, err := e.NewDMap("bucket-of-arbitrary-items")
  if err != nil {
    log.Fatalf("olric.NewDMap returned an error: %v", err)
  }

  ctx, cancel = context.WithCancel(context.Background())

  // Magic starts here!
  fmt.Println("##")
  fmt.Println("Simple Put/Get on a DMap instance:")
  err = dm.Put(ctx, "my-key", "Olric Rocks!")
  if err != nil {
    log.Fatalf("Failed to call Put: %v", err)
  }
 
  **gr, err := dm.Get(ctx, "my-key")**
  
How to get **my-key**  of **bucket-of-arbitrary-items** from another  instance ?

Thank you In advance

**Comment by buraksezer** [comment, marked as answer]
Already answered in Discord.
